discovery/search-category.yaml

```diff
@@ -0,0 +1,305 @@
+openapi: 3.1.0
+info:
+  title: Product Search and Category APIs
```

Comment:
I would like to rename this section as "Search and Category APIs", since it will also contain Content Search and Autosuggest APIs.

Reply:
Sounds good!